code/modules/events/immovable_rod.dm

```diff
@@ -81,4 +83,13 @@ In my current plan for it, 'solid' will be defined as anything with density == 1
 			H.adjustBruteLoss(160)
 		if(clong.density || prob(10))
 			clong.ex_act(2)
-	return
\ No newline at end of file
+
+/obj/effect/immovablerod/event
+	var/tiles_moved = 0
+
+/obj/effect/immovablerod/event/Move()
+	var/atom/oldloc = loc
+	. = ..()
+	tiles_moved++
+	if(get_dist(oldloc, loc) > 2 && tiles_moved > 10) // We went on a journey, commit sudoku
```

Comment:
Right. I thought about it further and figured that was likely the case, but I wasn't 100% sure. Thanks for clarifying.